### PR TITLE
Bounding box for precision -1

### PR DESCRIPTION
**Pull request by mridulnagpal**
Issue #107 
screenshot from 2018-01-04 04-29-32

### CODE REVIEW

**Comment by mridulnagpal** [comment]
@jywarren Please have a look, seems to work on my remote.

**Comment by jywarren** [comment]
Cool! Hm, what's the error here?

File: src/blurredLocation.js [hunk 1]
```diff
@@ -218,14 +218,15 @@ BlurredLocation = function BlurredLocation(options) {
...
+      else { bounds[1][0] = 1*(0.1**getPrecision()); }
     }
     if (!bounds[1][1]) {
-      if (getFullLon() < 0) { bounds[0][1] = -1; bounds[1][1] = 0; }
-      else { bounds[1][1] = 1; }
+      if (getFullLon() < 0) { bounds[0][1] = -1*(0.1**getPrecision()); bounds[1][1] = 0; }
+      else { bounds[1][1] = 1*(0.1**getPrecision()); }
     }
     if (rectangle) rectangle.remove();
+    console.log(bounds)
```
Comment (jywarren) [inline src/blurredLocation.js]: Oops, let's trim this out too! Thanks!

Comment (mridulnagpal) [reply src/blurredLocation.js]: Yeah sure, think I overlooked that. Thanks!

**Comment by mridulnagpal** [comment]
Corrected the issues and build should be passing now.

**Pull request merged**